src/librbd/ExclusiveLock.h

```diff
@@ -18,7 +19,9 @@ class ExclusiveLock : public ManagedLock<ImageCtxT> {
 
   ExclusiveLock(ImageCtxT &image_ctx);
 
-  bool accept_requests(int *ret_val = nullptr) const;
+  bool accept_requests() const;
```

Comment:
Nit: can we drop this and just update `librbd::Operations` to call the new method w/ the correct enum?